### DISCUSSION TITLE
availability of CI_Citation XML fragments

### DISCUSSION THREAD

**Discussion by kodebaets** [INSPIRE-MIF/helpdesk#140] [Q&A] [answered]
The 'TG for implementing dataset and service metadata' mentions that "_XML fragment documents containing the CI_Citation elements for the INSPIRE legal acts, technical guidance documents and conformance classes will be made available in the http://inspire.ec.europa.eu/id/citation/ namespace. References to these CI_Citation fragment documents may be used to encode the specification element by reference, using the xlink:href attribute of the gmd:specification element"_

Are there plans to make these xml fragments available? In particular inspire.ec.europa.eu/id/citation/ir/reg-1089-2010, to which some of our metadata contributors refer?

**Comment by fabiovinci** [comment, marked as answer]
Dear @kodebaets,

as indicated in the note in the section [4.2.2.1 Conformity](https://github.com/INSPIRE-MIF/technical-guidelines/blob/main/metadata/metadata-iso19139/metadata-iso19139.adoc#4221-conformity) of the MD TG there are no plans, at the moment, to provide pre-defined XML fragments. 

![image](https://user-images.githubusercontent.com/22212406/209880496-c7ff4324-1a3a-45d5-90c9-afc5792958a6.png)

The provision and use of XML fragments need to be evaluated since, as mentioned in some notes (e.g. NOTE 2 of the section [2.4.1. Conformity](https://github.com/INSPIRE-MIF/technical-guidelines/blob/main/metadata/metadata-iso19139/metadata-iso19139.adoc#241-conformity)) the citation of a document using this encoding could be incompatible with some clients.

![image](https://user-images.githubusercontent.com/22212406/209881059-9a77ff30-7399-4458-b54b-ad434c6eab29.png)

In addition, for Regulation 1089/2010 (with the proposed id: inspire.ec.europa.eu/id/citation/ir/reg-1089-2010) there could be a potential issue in providing a unique identifier since the XML fragment should contain the declaration of conformity which can assume different values.

![image](https://user-images.githubusercontent.com/22212406/209883143-7dd7f7e0-23ba-4cbe-a49a-1b7503d8c0f5.png)
